utils/atheepmgr/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2021 Sergey Ryazanov <ryazanov.s.a@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atheepmgr
+PKG_VERSION:=2.1.1
+PKG_RELEASE:=1
```

Comment:
normally you add a free line after PKG_RELEASE

Reply:
Will keep it in mind.

Reply:
Please change that. When the buildpr will be restarted, too.

We have a reference file how it should look like:
https://openwrt.org/docs/guide-developer/packages